osu.Game/Overlays/Direct/PlayButton.cs

```diff
@@ -130,15 +134,20 @@ private void updatePreviewTrack(bool playing)
 
                 Preview.Seek(0);
                 Preview.Start();

```

Comment:
Those two lines equal `Preview.Restart();`

Reply:
i can change that, but it's not part of this PR in the first place ;)